karaf-features/src/main/resources/features.xml

```diff
@@ -6,12 +6,22 @@
         <feature dependency="true">shell</feature>
         <feature version="${opennms.api.version}" dependency="true">opennms-integration-api</feature>
         <feature version="${project.version}">opennms-plugins-influxdb-client</feature>
+        <feature version="${resilience4j.version}" dependency="true">opennms-plugins-influxdb-resilience4j</feature>
         <bundle dependency="true">mvn:com.google.guava/guava/${guava.version}</bundle>
+        <bundle dependency="true">mvn:io.dropwizard.metrics/metrics-core/${metrics.version}</bundle>
         <bundle>mvn:org.opennms.plugins.timeseries/influxdb-plugin/${project.version}</bundle>
     </feature>
 
     <feature name="opennms-plugins-influxdb-client" version="${project.version}">
         <bundle>mvn:org.opennms.plugin.timeseries.influxdb.wrap/influxdb-client/${project.version}</bundle>
     </feature>
 
+    <feature name="opennms-plugins-influxdb-resilience4j" description="resilience4j" version="${resilience4j.version}">
```

Comment:
resilience4j has problems when loading multiple versions from OSGi - create a wrap module like we did here to avoid these: https://github.com/OpenNMS/opennms-cortex-tss-plugin/tree/master/wrap

Reply:
fixed.